src/main/java/com/mg105/controllers/TutorialTextController.java

```diff
@@ -2,13 +2,15 @@
 
 import com.mg105.entities.GiveTutorial;
 import com.mg105.use_cases.PlayerGetsTutorial;
-import com.mg105.utils.TutorialTexts;
+import com.mg105.entities.items.utils.TutorialTexts;
```

Comment:
This isn't the correct package path. The path should be com.mg105.utils. Moved the utils folder to there.

This appears in multiple places (like all the item type classes). If you move the package/folder it should probably change it automatically.